src/python/CRABInterface/DataWorkflow.py

```diff
@@ -168,12 +167,10 @@ def submit(self, workflow, activity, jobtype, jobsw, jobarch, use_parent, second
             collector = backend_urls['htcondorPool']
 
         try:
-            requestname = '%s:%s_%s' % (timestamp, userhn, workflow)
+            taskname = '%s:%s_%s' % (timestamp, userhn, workflow)
             schedd_name = self.chooseScheduler(scheddname, backend_urls).split(":")[0]
-            if len(requestname) > RX_WORKFLOW_LEN:
-              raise InvalidParameter("Taskname exceed maximum allowed lenght, please review parameter requestName in your config file")
```

Comment:
Yes, let's keep a check on final taskname as well. PLEASE !


On 11/14/2015 01:18 PM, Justas Balčas wrote:
> In src/python/CRABInterface/DataWorkflow.py
> <https://github.com/dmwm/CRABServer/pull/5078#discussion_r44857967>:
>
>>              schedd_name = self.chooseScheduler(scheddname, backend_urls).split(":")[0]
>> -            if len(requestname) > RX_WORKFLOW_LEN:
>> -              raise InvalidParameter("Taskname exceed maximum allowed lenght, please review parameter requestName in your config file")
>
> It is easy to remove code, but not to add. Personally, I would leave validation also of final
> taskname, in case cern would think they need to increase username len and we just might miss it.
> Also validation should be done through WMCore validate_str
>